tools/src/main/java/org/bitcoinj/tools/WalletTool.java

```diff
@@ -761,7 +761,7 @@ public void onCoinsSent(Wallet wallet, Transaction tx, Coin prevBalance,
             case BLOCK:
                 peers.addDataEventListener(new AbstractPeerDataEventListener() {
                     @Override
-                    public void onBlocksDownloaded(Peer peer, Block block, @Nullable FilteredBlock filteredBlock, int blocksLeft) {
+                    public void onBlocksDownloaded(Peer peer, AbstractBlockHeader block, int blocksLeft) {
```

Comment:
The entire parameter is dropped. A major part of what this does is to clean up the API so Block and FilteredBlock have a common superclass.